### DISCUSSION TITLE
Semantic Kernel memory vs Kernel Memory

### DISCUSSION THREAD

**Discussion by John-Bell** [microsoft/semantic-kernel#6197] [General]
I’ve been trying to use Pinecone with the copilot chat code and run into a confusion of KM vs SKM and which one has a future.  Worth noting that the Pinecone connector is no longer compatible since Pinecone have made breaking changes to their API.
My question is, are both experimental or os one here to stay?
Thanks in advance

**Comment by sophialagerkranspandey** [comment]
Semantic Kernel Memory should become non-experimental soon as we're looking through and fixing features like this. For Kernel memory, this will remain experimental due to it's nature of being an experimental project in Microsoft.

**Reply by John-Bell** [reply]
Thank you so much for taking the time to respond
